_episodes_rmd/01-introduction.Rmd

```diff
@@ -101,14 +101,17 @@ standalone software.
 
 ## Log on to RStudio Server
 
-Open a web browser and enter the IP address of your instance, followed by
+Open a web browser and enter the IP address of your instance 
+(provided by your instructors), followed by
 `:8787`. For example, if your IP address was 123.456.789 your URL would be
```

Comment:
I think we should switch this to be more like a real IP address with 4 number groupings -- 123.45.67.89 (and also below)

Reply:
Good catch. Fixed!